.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+<!--
+Thank you for your Pull Request. Please provide a description above and review
+the requirements below.
+
+Bug fixes and new features should include tests.
+
+Contributors guide: https://github.com/MobileTeleSystems/RecTools/blob/master/CONTRIBUTING.rst
+-->
+
+## Description
+
+<!--
+Explain the context and why you're making the change. What is the problem
+you're trying to solve?
+-->
+
+## Type of change
+
+<!--
+Please delete/mark options that are/aren't relevant
+-->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Optimization
+
+## How Has This Been Tested?
```

Comment:
Can we change this to checklist here? Seems very useful.

- Did you read the contribution guide?
- Did you update the documentation strings for public classes and methods? Please use Numpy format
- Did you write all the necessary tests?
- Did you update the changelog?

Reply:
Also:
- Did you update docs?
- Did you update examples?

Reply:
This makes sense, but I can't help but wonder -- doesn't it belong to the contributing.md file? My assumption is that one reads it before pulling the project, changing it and actually opening a PR. That's what I would do. But I might be expecting too much of contributors.
Wdyt?

Reply:
I believe checklist to be the most convenient way to remind all the contributors (including ourselves) to do the basic stuff with every PR. Like updating changelog for example. So this is a good place.
I've started working on updating contributing guide as well.

Reply:
No worries, I'll add it shortly